src/frontend/Komponenter/Behandling/Høyremeny/BehandlingHistorikk.tsx

```diff
@@ -20,6 +21,9 @@ const BehandlingHistorikk: React.FC<IBehandlingHistorikkProps> = ({ behandlingId
     return (
         <DataViewer response={{ behandlingHistorikkResponse: behandlingHistorikk, behandling }}>
             {({ behandlingHistorikkResponse, behandling }) => {
+                const skalViseBegrunnelse = !behandlingHarBlittUnderkjentDeretterGodkjent(
```

Comment:
For å unngå `!` (som jeg selv bruker en del selv også), er det tilstrekkelig å kun sjekke om det **ikke** finnes en 
`VEDTAK_GODKJENT` ?

Reply:
Tror det kunne vært lurt, ja 👍